tools/depends/target/waylandpp/Makefile

```diff
@@ -0,0 +1,54 @@
+-include ../../Makefile.include
+DEPS=Makefile
+
+# lib name, version
+LIBNAME=waylandpp
+VERSION=0.1.0
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.gz
+
+LIBDYLIB=$(PLATFORM)/src/libwayland-client++.so
+CLEAN_FILES=$(ARCHIVE) $(PLATFORM)
+
+ifeq ($(PLATFORM),)
+	# Building stand-alone
+	CXX_FOR_BUILD := $(CXX)
+	ROOT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+	PLATFORM = native
+	TARBALLS_LOCATION = $(ROOT_DIR)
+	BASE_URL := http://mirrors.kodi.tv/build-deps/sources
+	RETRIEVE_TOOL := curl -Ls --create-dirs -f -O
+	ARCHIVE_TOOL := tar --strip-components=1 -xf
+else
+	# Building as part of depends
+	DEPS += ../../Makefile.include
+	export PKG_CONFIG=$(NATIVEPREFIX)/bin/pkg-config
+	export PKG_CONFIG_LIBDIR=$(PREFIX)/lib/pkgconfig
+endif
+
+SCONS=scons
+export CROSSCXX:=$(CXX)
+export CXX=$(CXX_FOR_BUILD)
+
+all: .installed-$(PLATFORM)
+
+$(TARBALLS_LOCATION)/$(ARCHIVE):
+	cd $(TARBALLS_LOCATION); $(RETRIEVE_TOOL) $(RETRIEVE_TOOL_FLAGS) $(BASE_URL)/$(ARCHIVE)
+
+$(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
```

Comment:
something like https://github.com/xbmc/xbmc/blob/94faa07/tools/depends/target/crossguid/Makefile#L48-L52 is missing